README.md

```diff
@@ -44,7 +44,7 @@ The following are also required:
   
 #### Terraform Requirements:
 
-- [Terraform](https://www.terraform.io/downloads.html) v1.0.0
+- [Terraform](https://www.terraform.io/downloads.html) v1.1.6
```

Comment:
is this correct? I just saw something where you were switching back to Terraform 1.0.0

Reply:
Fixed it back to 1.0.0